src/parseTools.js

```diff
@@ -480,7 +490,7 @@ function checkSafeHeap() {
 }
 
 function getHeapOffset(offset, type) {
-  if (Runtime.getNativeFieldSize(type) > 4 && type == 'i64') {
+  if (!WASM_BIGINT && Runtime.getNativeFieldSize(type) > 4 && type == 'i64') {
```

Comment:
Are we sure this will be no-op for wasm32 users of `WASM_BIGINT`?

Reply:
according to the test, yes :)